tests/test_service_account.py

```diff
@@ -103,7 +102,9 @@ def _from_json_keyfile_name_helper(payload, scopes=None,
         try:
             with open(filename, 'w') as file_obj:
                 json.dump(payload, file_obj)
-            return ServiceAccountCredentials.from_json_keyfile_name(
+            # TODO(issue 564): Cleanup Credentials access
```

Comment:
For some reason, that style of wrapping escaped me; I was trying that with parenthesis before and after the dot, but not without them.  Updating per your suggestion.